cmd/server-test/multi_analyzer_test.go

```diff
@@ -41,8 +41,7 @@ func (suite *MultiDummyIntegrationSuite) TestSuccessReview() {
 	suite.GrepAll(suite.r, []string{
 		"processing pull request",
 		"posting analysis",
-		`{"analyzer-name":"Dummy1","file":"another.go","line":3,"text":"This line exceeded`,
-		`{"analyzer-name":"Dummy2","file":"another.go","line":3,"text":"This line exceeded`,
+		`"file":"another.go","line":3,"text":"This line exceeded`,
```

Comment:
This test checks if lookout works correctly with more than 1 analyzer. Now it's not clear if the second analyzer works.

Reply:
You're right! I'm gonna fix it, thanks! 👍